src/vm/docs/mod.rs

```diff
@@ -1346,14 +1346,20 @@ const GET_BLOCK_INFO_API: SpecialAPI = SpecialAPI {
     description: "The `get-block-info?` function fetches data for a block of the given block height. The
```

Comment:
I think you should clarify that the block height input might refer to either the burn chain or the. stacks blockchain, depending on the property

Reply:
Aaron suggested making a new function called "get-burn-block-info?" instead.

Do you think it would be easier to make the docs if we did that?

https://github.com/blockstack/stacks-blockchain/issues/2663#issuecomment-866154516

Reply:
Oh - yes, I definitely prefer the approach of putting this functionality into a separate clarity function, I think that is way less prone to user error

Reply:
Yes. In the future `get-burn-block-info?` could also take `stacks-tip` and return the stacks tip.

Reply:
Did this.. turned out to be a big delta but worth it!